src/components/general/Animation.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import { Animated, Easing, Dimensions } from "react-native";
+
+const dimension = Dimensions.get("screen");
+
+/**
+ * @classdesc Component to render different types of animations with the timing() function.
+ */
+
+class Animation extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = { animated: new Animated.Value(
+			this.props.initial.ratio * dimension[this.props.initial.screen]
+			|| this.props.initial
+		) };
+	}
+
+	timing(animatedObject, callback) {
+		Animated.timing(this.state.animated, animatedObject).start(() => callback && callback());
+	}
+
+	render() {
+		const { containerStyle, children, type, property } = this.props;
+
+		switch (type) {
+			case "View":
+				return (
+					<Animated.View style = { [containerStyle, { [property]: this.state.animated }] }>
+						{ children }
+					</Animated.View>
+				);
+		}
+	}
+}
+
+/**
+ * @classdesc Utility class for Animation component. Used to start a transition with the Animation class and
+ *            static sliding() function.
+ */
+
+export class Animate {
+	/**
+	 * @description Sliding animation component that animates a View based on the static sliding() function.
+	 *
+	 * @param {string}          type           Type of Animation. ["View"]
+	 * @param {Object}          containerStyle Style of Animation component.
```

Comment:
Can you add a 2 space wide gap between each column of comments. It looks like a sentence if you don't.